ui/src/app/cohort-search/cohort-search/cohort-search.component.spec.ts

```diff
@@ -150,7 +153,6 @@ describe('CohortSearchComponent', () => {
   }));
 
   beforeEach(() => {
-    registerApiClient(CohortsApi, new CohortsApiStub());
```

Comment:
This line seemed correct to me since the component uses the cohortsApi. I'd suggest leaving this even if the current test coverage doesn't actually hit the lines that require this.

Reply:
Oops, sorry - I thought these were two different tests - carry on